src/languages/ebnf.js

```diff
@@ -9,7 +9,7 @@ function(hljs) {
 
     var nonTerminalMode = {
         className: "attribute",
-        begin: /^[ ]*[a-zA-Z][a-zA-Z-]*([\s-]+[a-zA-Z][a-zA-Z]*)*/
+        begin: /^[ ]*[a-zA-Z][a-zA-Z-_]*([\s-_]+[a-zA-Z][a-zA-Z]*)*/
```

Comment:
I know you didn't add \s but is a space truly allowed in an attribute name? How does that work?

Having to add the _ in two places is what got my attention here and caused me to notice this.

Reply:
It's not necessarily common practice, but spaces are consumed in meta identifiers and the parts concatenated. There's some discussion here about it: https://stackoverflow.com/questions/15268229/ebnf-whitespacing-in-meta-identifiers

Reply:
Also examples on the Wikipedia page. https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form

Reply:
Thanks, from what I'm reading this could probably be simplified, but this will work for now. ;-)